maven/archetypes/vra-ng/src/main/resources/archetype-resources/content.yaml

```diff
@@ -39,3 +43,5 @@ resource-action:
 catalog-entitlement:
 content-source:
 property-group:
+policy:
+  content-sharing:
```

Comment:
Can you please add the same here: https://github.com/vmware/build-tools-for-vmware-aria/blob/main/maven/archetypes/ts-vra-ng/src/main/resources/archetype-resources/vra/content.yaml

Reply:
I have made the changes in the file you specified above.